### PR TITLE
docs: add quickstart for spin application

### PR DESCRIPTION
**Pull request by zr-msft**
Signed-off-by: Zach Rhoads <zachary.rhoads@microsoft.com>

### CODE REVIEW

File: containerd-shim-spin-v1/quickstart.md [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+HTTP base: /
+HTTP path: /hi
+```
+
+## Build the application
+
+Navigate to the directory where you created the application:
+
+```bash
+cd qs-wasm-spin-shim
```
Comment (devigned) [inline containerd-shim-spin-v1/quickstart.md:87]:
```suggestion
cd qs-wasm-spin
```

I believe there might be a mistake with this directory.

**Review by devigned — CHANGES_REQUESTED** [review]
Nice work! Looks pretty solid. Might have a directory issue though.

**Comment by zr-msft** [comment]
Thanks @devigned, nice catch! Added your change

File: containerd-shim-spin-v1/quickstart.md [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+
+## Pre-requisites
+Before you begin, you need to have the following installed:
+
+- [Docker](https://docs.docker.com/install/)
+- [k3d](https://k3d.io/v5.4.6/#installation)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+- [Spin binary and templates](https://spin.fermyon.dev/quickstart/)
+- [Rust](https://www.rust-lang.org/tools/install)
+-
```
Comment (zr-msft) [inline containerd-shim-spin-v1/quickstart.md:11]:
```suggestion
```
typo fix

**Review by devigned — APPROVED** [review]
lgtm

**Comment by zr-msft** [comment]
Thanks @devigned 

Do you mind merging?

**Pull request merged**